data/releases/schedule.yaml

```diff
@@ -5,12 +5,17 @@ schedules:
 - release: 1.28
   releaseDate: 2023-08-15
   next:
-    release: 1.28.4
+    release: 1.28.5
     cherryPickDeadline: 2023-12-01
     targetDate: 2023-12-06
   maintenanceModeStartDate: 2024-08-28
   endOfLifeDate: 2024-10-28
   previousPatches:
+    - release: 1.28.4
+      cherryPickDeadline: ""
+      targetDate: 2023-11-14
+      note: >-
+        Out of band release to fix [CVE-2023-5528](https://github.com/advisories/GHSA-hq6q-c2x6-hmch)
```

Comment:
Let's link to our official announcement in this file too: https://groups.google.com/g/kubernetes-announce/c/c3py6Fw0DTI/m/cScFSdk1BwAJ